.github/workflows/generate-python-driver.yml

```diff
@@ -0,0 +1,74 @@
+on: [ workflow_call ]
+
+env:
+  SPEC_DIR: ${{ github.workspace }}/specs_preprocessed
+  PYTHON_DIR: ${{ github.workspace }}/sdks/python
+
+jobs:
+  setup-environment:
+    name: Setup Environment
+    runs-on: ubuntu-latest
+    env:
+      ORIG_SPEC_DIR: ${{ github.workspace }}/res
+      PREPROCESS: true
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Python
+        if: env.PREPROCESS == 'true'
+        run: |
+          apt-get update
+          apt-get install -y python3 python3-pip
+          python3 -m pip install --upgrade pip
+          python3 -m pip install -r $PYTHON_DIR/processing/requirements.txt
+
+      - name: Preprocess OpenAPI specs
+        if: env.PREPROCESS == 'true'
+        id: preprocess
+        run: |
+          python3 $PYTHON_DIR/processing/spec_preprocessing.py $ORIG_SPEC_DIR $SPEC_DIR
+          echo "SPEC_DIR=$SPEC_DIR" >> $GITHUB_ENV
+
+  generate-python-driver:
+    name: Test Python drivers
+    runs-on: ubuntu-latest
+    needs: setup-environment
+    container:
+      image: openapitools/openapi-generator-cli:latest
+    strategy:
+      max-parallel: 8
+      matrix:
+        driver: [
```

Comment:
There's also `vee-apis` now. But the bigger issue is there are too many files. Can we consolidate for the purposes of SDK generation if we can't combine the files themselves in the vserver repo?

Reply:
Yes, it can be done. We can create a unified specification with references to the individual ones using $ref.
We need to decide how to combine them. 
But this is beyond the scope of this task.